__tests__/tests.js

```diff
@@ -0,0 +1,146 @@
+let knex = require('./../database/index.js');
+let { db, Listing } = require('./../database/mongo.js');
+let generateBookedDates = require('./../database/generateBookedDates')
+
+const randomNumberUpTo = (limit) => Math.floor(Math.random() * limit);
+
+jest.setTimeout(60000);
+
+let testIterations = 1000;
+
+describe('PostgresQL Query Speeds', () => {
+  
+  test('fetching listing and bookeddates takes <= 50ms', async () => {
+
+    let totalFetchListingTimeElapsed = 0;
+    let totalFetchBookedDatesTimeElapsed = 0;
+
+    for (let i = 0; i < testIterations; i++) {
+      let id = randomNumberUpTo(10000000);
+
+      let t1 = Date.now();
+      let listing = await knex.select().from('bookings.listings').where('id', id).limit(1);
```

Comment:
I'm curious if the `select` statement would be any faster using raw SQL. Looks like there is a pending issue to this effect:
https://github.com/tgriesser/knex/issues/2518

I'm also seeing the examples in the knex docs using `select('*')`, which maybe is faster since it doesn't have to compute the `*`. Probably not a big impact though.

Reply:
Yes, turns out using raw SQL is faster. I am refactoring my API to use it.